hexa/pipelines/schema.py

```diff
@@ -270,13 +271,22 @@ def resolve_create_pipeline(_, info, **kwargs):
         }
 
     try:
+        if input.get("timeout") and input.get("timeout") > int(
+            os.environ.get("PIPELINE_RUN_MAX_TIMEOUT")
+        ):
+            raise Exception("Pipeline timeout greater than the maximum allowed value.")
```

Comment:
I think we should return a specific error code for this, that the SDK can understad